src/accounts/accounts.controller.ts

```diff
@@ -0,0 +1,60 @@
+import { Body, Controller, Get, Param, Post } from '@nestjs/common';
+import { AccountsService } from './accounts.service';
+import { AccountDto } from './account.dto';
+import { TransactionsDto } from '../transactions/transactions.dto';
+import {
+  AccountValidationPipe,
+  CreateAccountValidationPipe,
+  DepositValidationPipe,
+  WithdrawValidationPipe,
+  SendValidationPipe,
+} from './accountValidationPipe';
+
+@Controller('accounts')
+export class AccountsController {
+  constructor(private accountService: AccountsService) {}
+
+  @Get()
+  public getAccounts() {
+    return this.accountService.getAccounts();
+  }
+
+  @Get(':id')
```

Comment:
If you have an id the path should indicate what type of id it is. Eg. accounts/:id